compiler/passes/resolveIntents.cpp

```diff
@@ -131,13 +131,13 @@ IntentTag blankIntentForType(Type* t) {
     // allow blank intent owned to be transferred out of
     retval = INTENT_CONST_REF;
 
-  } else if (is_bool_type(t)                         ||
-             is_int_type(t)                          ||
-             is_uint_type(t)                         ||
-             is_real_type(t)                         ||
-             is_imag_type(t)                         ||
-             is_complex_type(t)                      ||
-             is_enum_type(t)                         ||
+  } else if (isBoolType(t)                         ||
```

Comment:
ultra-nit: this mucked up the `||` (as well as `\` for macro definitions, but I won't ask you to replace them 😄

Reply:
But that will bug me, so I will fix them :)